src/System.Data.Common/tests/System/Data/Common/DbCommandTest.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
```

Comment:
This is missing a line. It should be:
```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

Reply:
Ah... was this file just moved?  Looks like it may have originally come from Mono.  You can leave the header as-is then, though we probably need to add something additional to it subsequently.

Reply:
Yes, this was just a rename. Note that most (all?) other test files have the similar two-line version - let me know if something needs to be done on this.